[2024-10-16T09:40:08.133+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-09-05T00:00:00+00:00 [queued]>
[2024-10-16T09:40:08.162+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-09-05T00:00:00+00:00 [queued]>
[2024-10-16T09:40:08.165+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:40:08.168+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:40:08.172+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:40:08.218+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-09-05 00:00:00+00:00
[2024-10-16T09:40:08.230+0000] {standard_task_runner.py:55} INFO - Started process 4234 to run task
[2024-10-16T09:40:08.240+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_rus_dag', 'insert_rate', 'scheduled__2024-09-05T00:00:00+00:00', '--job-id', '366', '--raw', '--subdir', 'DAGS_FOLDER/RUS_DAG.py', '--cfg-path', '/tmp/tmp9incc9fd']
[2024-10-16T09:40:08.242+0000] {standard_task_runner.py:83} INFO - Job 366: Subtask insert_rate
[2024-10-16T09:40:08.523+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-09-05T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:40:08.672+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:40:08.858+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_rus_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-09-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-09-05T00:00:00+00:00
[2024-10-16T09:40:08.862+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUS', None, '2024-09-05')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:40:08.895+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:40:09.716+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:40:09.740+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUS', None, '2024-09-05')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:40:09.748+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 261, in execute
    return_last=self.return_last,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 338, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 369, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedColumn: column "none" does not exist
LINE 3: VALUES ('USD', 'RUS', None, '2024-09-05')
                              ^

[2024-10-16T09:40:09.776+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=exchange_rate_usd_rus_dag, task_id=insert_rate, execution_date=20240905T000000, start_date=20241016T094008, end_date=20241016T094009
[2024-10-16T09:40:09.897+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 366 for task insert_rate (column "none" does not exist
LINE 3: VALUES ('USD', 'RUS', None, '2024-09-05')
                              ^
; 4234)
[2024-10-16T09:40:09.932+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2024-10-16T09:40:09.977+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T09:49:55.953+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-09-05T00:00:00+00:00 [queued]>
[2024-10-16T09:49:55.970+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-09-05T00:00:00+00:00 [queued]>
[2024-10-16T09:49:55.971+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:49:55.971+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:49:55.972+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:49:55.995+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-09-05 00:00:00+00:00
[2024-10-16T09:49:56.002+0000] {standard_task_runner.py:55} INFO - Started process 7085 to run task
[2024-10-16T09:49:56.007+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_rus_dag', 'insert_rate', 'scheduled__2024-09-05T00:00:00+00:00', '--job-id', '1244', '--raw', '--subdir', 'DAGS_FOLDER/RUS_DAG.py', '--cfg-path', '/tmp/tmp5siwuexm']
[2024-10-16T09:49:56.009+0000] {standard_task_runner.py:83} INFO - Job 1244: Subtask insert_rate
[2024-10-16T09:49:56.113+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-09-05T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:49:56.173+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:49:56.256+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_rus_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-09-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-09-05T00:00:00+00:00
[2024-10-16T09:49:56.257+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUB', 89.49829813, '2024-09-05')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:49:56.270+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:49:56.542+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:49:56.552+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUB', 89.49829813, '2024-09-05')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:49:56.555+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T09:49:56.575+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_rus_dag, task_id=insert_rate, execution_date=20240905T000000, start_date=20241016T094955, end_date=20241016T094956
[2024-10-16T09:49:56.596+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T09:49:56.620+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
